### PR TITLE
Add prow config for knative/sample-source

### PR DESCRIPTION
**Pull request by lberk**
**What this PR does, why we need it**:
Needed prow config for [knative/sample-source](https://github.com/knative/sample-source)

**Which issue(s) this PR fixes**:
Fixes #

**Special notes to reviewers**:
@chizhg PR as requested on slack.
**User-visible changes in this PR**:

### CODE REVIEW

File: ci/prow/config_knative.yaml [hunk 1]
```diff
@@ -177,6 +177,12 @@ presubmits:
     - unit-tests: true
       dot-dev: true
 
+  knative/sample-source:
```
Comment (chizhg) [inline ci/prow/config_knative.yaml]: Please also add CI jobs for it, as what we have for `sample-controller` - https://github.com/knative/test-infra/blob/2c4cd9df33c648a86d95bdd4119386cba04cde56/ci/prow/config_knative.yaml#L416.

It does not hurt for not having it though, but it's better to follow the same pattern.

Comment (lberk) [reply ci/prow/config_knative.yaml]: Ah sorry, thought that might've been what was meant by the guide saying to not setup presubmit tests just yet.  Will add it now.

Comment (lberk) [reply ci/prow/config_knative.yaml]: added in a80e114

**Pull request merged**